nms/packages/magmalte/app/views/equipment/EnodebDetailConfig.js

```diff
@@ -34,9 +34,826 @@ import {colors, typography} from '../../theme/default';
+      title: 'Index',
+      width: 120,
+      render: function (o) {
+        return o.index - 1;
+      },
+    },
+    {
+      field: 'enable',
+      title: 'Enable',
+      width: 150,
+    },
+    {
+      field: 'plmn',
+      title: 'PLMN',
+      width: 150,
+    },
+    {
+      field: 'cellId',
+      title: 'Cell ID',
+      minWidth: 120,
+    },
+    {
+      field: 'earfcn',
+      title: 'EARFCN',
+      width: 150,
+    },
+    {
+      field: 'pci',
+      title: 'PCI',
+      width: 150,
+    },
+    {
+      field: 'qOffset',
+      title: 'qOffset',
+      width: 150,
+    },
+    {
+      field: 'cio',
+      title: 'CIO',
+      width: 150,
+    },
+    {
+      field: 'tac',
+      title: 'TAC',
+      width: 150,
+    },
+    {
+      field: 'op',
+      title: 'Operations',
+      width: 150,
+      render: function (o) {
+        return (
+          <>
+            <IconButton aria-label="delete" size="small">
+              <DeleteIcon
+                onClick={() => {
+                  try {
+                    remove(o, function (p) {
+                      const list = JSON.parse(JSON.stringify(p));
+                      try {
+                        list.map(function (item) {
+                          [
+                            'index',
+                            'cellId',
+                            'earfcn',
+                            'pci',
+                            'qOffset',
+                            'cio',
+                            'tac',
+                          ].map(function (key) {
+                            item[key] = parseInt(item[key]);
+                          });
+
+                          delete item.tableData;
+                        });
+                      } catch (e) {}
+                      enbInfo.enb.enodeb_config.managed_config.NeighborCellList = list;
+                      enbInfo.enb.config.NeighborCellList = list;
+                      ctx.setState(enbInfo.enb.serial, {
+                        ...enbInfo,
+                        enb: enbInfo.enb,
+                      });
+                      enqueueSnackbar('eNodeb deleted successfully', {
+                        variant: 'success',
+                      });
+                    });
+                  } catch (e) {}
+                }}
+                fontSize="inherit"
+              />
+            </IconButton>
+          </>
+        );
+      },
+    },
+  ];
+
+  return (
+    <div>
+      <div>
+        Neigh Cell List
+        <IconButton aria-label="add" size="small">
+          <CellFormDialog
+            list={enbInfo.enb.config.NeighborCellList}
+            onSave={cell => {
+              try {
+                addCell(cell, function (p) {
+                  const list = JSON.parse(JSON.stringify(p));
+                  try {
+                    list.map(function (item) {
+                      [
+                        'index',
+                        // 'plmn',
```

Comment:
Remove this commented out line, otherwise add more comment and/or a TODO if it's planning to be added back. Also note a github username if you add a TODO, or reference a Github issue number.

Reply:
delete the comments .